deployments/rialto/Front-end.Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM node:12 as build-deps
+
+# install tools and dependencies
+RUN set -eux; \
+	apt-get install -y git
+
+# clone UI repo
+RUN cd /usr/src/ && git clone https://github.com/paritytech/bridge-ui.git
+WORKDIR /usr/src/bridge-ui
+RUN yarn
+ARG SUBSTRATE_PROVIDER
+ARG ETHEREUM_PROVIDER
+ARG EXPECTED_ETHEREUM_NETWORK_ID
+
+ENV SUBSTRATE_PROVIDER $SUBSTRATE_PROVIDER
+ENV ETHEREUM_PROVIDER $ETHEREUM_PROVIDER
+ENV EXPECTED_ETHEREUM_NETWORK_ID $EXPECTED_ETHEREUM_NETWORK_ID
```

Comment:
I removed these and everything still worked, what are they being used for right now?

Reply:
When you say it still worked, you mean it still connected to the local nodes (the Subsrate one I mean because the Ehtereum Env variable is actually only used in the Error message for users to point their Metamask).

The `ENV $ARG` is to my knowledge the only way to pass any dynamic value form docker-compose to a Docker image. I've seen that in several places I think, more notably 
https://vsupalov.com/docker-arg-env-variable-guide/#setting-env-values